RELEASE.md

```diff
@@ -1,27 +1,52 @@
 # Changelog
 
 > Contains description of QHub releases.
----
-## To create a new release:
 
-1. Increment the version number in `qhub/VERSION`
-2. Ensure that the version number in `qhub/VERSION` is used in pinning QHub in the github actions
-`qhub/template/{{ cookiecutter.repo_directory }}/.github/workflows/qhub-ops.yaml`
-
----
+______________________________________________________________________
 
 ## Upcoming Release
 
 ### Feature changes and enhancements
 
 ### Bug fixes
 
-## Release v0.4.0 - 03/17/2022
+## Release v0.4.0 - March 17, 2022
+
+**WARNING**
+> If you're looking for a stable version of QHub, please consider `v0.3.14`. The `v0.4.0` has many breaking changes and has several bugs that we are working to resolve in the upcoming releases.
```

Comment:
change to something like: `has rough edges that we will be smoothing over in upcoming point releases`